client/ayon_shotgrid/addon.py

```diff
@@ -113,14 +121,6 @@ def create_shotgrid_session(self):
                 "password": sg_password
             })
 
-        elif self._client_login_type == "tray_api_key":
-            sg_username, _ = credentials.get_local_login()
-            kwargs.update({
-                "username": sg_username,
-                "api_key": self._shotgrid_api_key,
-                "script_name": self._shotgrid_script_name,
-            })
-
```

Comment:
is there any reason why this had been removed?

This is now resulting in following error

```
Traceback (most recent call last):
  File "C:\Users\jakub\AppData\Local\Ynput\AYON\dependency_packages\ayon_2407261707_windows.zip\dependencies\pyblish\plugin.py", line 528, in __explicit_process
    runner(*args)
  File "C:\CODE\__PYPE\ayon-shotgrid\client\ayon_shotgrid\plugins\publish\collect_shotgrid_session.py", line 23, in process
    raise KnownPublishError(
ayon_core.pipeline.publish.publish_plugins.KnownPublishError: User not found in environment, make sure it's set.
```

Reply:
Sorry, I missed this comment. 

> is there any reason why this had been removed?

It's not removed, it was moved. 
The `get_credentials` should return the username and password based on the login type. 
and the `create_shotgrid_session` should update the `kwargs` variable properly based on the login type. 

I wonder if this issue is related to the fallback that was removed. should we put it back 
https://github.com/ynput/ayon-shotgrid/pull/118/files#r1724742824 ?